### DISCUSSION TITLE
Changing "import path" - impact?

### DISCUSSION THREAD

**Discussion by DJF3** [immich-app/immich#8026] [Q&A]
Team,
I have 20 external libraries (NFS shares on NAS)
`/usr/src/app/mypics2010` (pointing to /volume1/PICTURES/2010 on NAS)
`/usr/src/app/mypics2011` (pointing to /volume1/PICTURES/2011 on NAS)
`/usr/src/app/mypics2012` (pointing to /volume1/PICTURES/2012 on NAS)
etc.
These are configured as 20 separate external libraries in Immich.

For technical reasons I have to change to 1 single NFS share "`/volume1/PICTURES`".
Given the amount of CPU power I've spent to analyse/thumbnail these pictures I would just like to **_change the image paths_** because nothing else changed. So....
**FROM**: `/usr/src/app/mypics2010`   **TO** `/mnt/naspics/2010` (pointing to /volume1/PICTURES/2010 on NAS)

Is that possible without having to do everything all over again (like generating thumbnails, recognizing objects/faces, etc)

Thank you!

**Comment by bo0tzz** [comment]
Changing the volume on the host shouldn't affect anything. Just leave the paths inside of the container the same.

**Reply by DJF3** [reply]
Well, before I had 20 _volumes_ mounted
**20 mounted volumes** -- `/usr/src/app/mypics20xx`  **pointing to** `/volume1/PICTURES/20xx`
   and 20 'import paths' pointing to /usr/src/app/mypics20xx folders.
  (was probably not ideal :-) )

Changing to 
**1 mounted volume**  -- `/mnt/naspics/`  **pointing to** `/volume1/PICTURES/`
   and 20 'import paths' pointing to `/mnt/naspics/20xx/` folders.

When changing the import path, the actual files seen in that location are _identical_ to the files seen before.

(btw that response was _FAST_! 👍 )
